testplan/testing/multitest/driver/sqlite.py

```diff
@@ -68,7 +78,19 @@ def __init__(self, name, db_name, connect_at_start=True, **options):
     @property
     def db_path(self):
```

Comment:
The db_path in config option is the dir portion of the db_path, but the db_path property is the full db_path. I don't think this is good.

Let's use just db_path in config option then, remove db_name.